Task12/JsonCatalogSerializer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.Json;
+
+namespace Task12
+{

```

Comment:
Please follow the task requirements:
7	Save catalog object into json files (**one file-one author with all his books.**)
8	Restore catalog object from json files. Check identity.